lib/routes/account.js

```diff
@@ -959,10 +980,13 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+          return db.sessionTokenVerified(sessionToken.id)
```

Comment:
IIUC, this approach will mean that we read the `sessionToken` from the db twice - once in the `sessionToken` auth strategy that guards this handler, and again here to check its verification status.  Perhaps we should eagerly load the `tokenVerified` property at auth-checking time, so that we can just refer to its value here without hitting the db again.

(This is the most frequently called endpoint in the entire stack, another db hit on this route will add up quick)